### DISCUSSION TITLE
[bugs]公式渲染问题

### DISCUSSION THREAD

**Discussion by kevinzhangcode** [HEIGE-PCloud/DoIt#518] [Q&A] [answered]
## 描述你遇到的错误
`typora`渲染正常
<img width="409" alt="image" src="https://user-images.githubusercontent.com/38588781/157789258-b8b43aed-30db-46fc-a874-e3f2094fa713.png">

`typora`源码
<img width="482" alt="image" src="https://user-images.githubusercontent.com/38588781/157789288-95759f5e-44ea-454c-853a-6901a5e0f91a.png">

DoIt渲染错误
<img width="598" alt="image" src="https://user-images.githubusercontent.com/38588781/157789041-8a120b75-33d8-4815-9546-3b82f761883f.png">

## 可能的问题

见https://github.com/dillonzq/LoveIt/issues/57

似乎是从LoveIT继承的问题

**Comment by HEIGE-PCloud** [comment, marked as answer]
用 math shortcode 来强制 escape 数学表达式。
https://hugodoit.pages.dev/zh-cn/theme-documentation-extended-shortcodes/#math

```
{{< math >}}${\#}x${{< /math >}}
{{< math >}}$\boldsymbol{{w}_{\#x}}${{< /math >}}
```

**Reply by HEIGE-PCloud** [reply]
Related discussion https://github.com/HEIGE-PCloud/DoIt/issues/126

**Reply by kevinzhangcode** [reply]
非常感谢🙏，已经解决了问题 😊@HEIGE-PCloud
